etc/release_manifests/EXAMPLE-ML-DGA-20201111-1.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
this should be only in GH releases, not included directly in the repo

Reply:
cc2ba1f88e80f6f1dcbb41890b32a3b703ac5e65